/rm_fire_control:
  ros__parameters:
    large_armor_width: 0.23
    small_armor_width: 0.135
    large_armor_height: 0.127
    small_armor_height: 0.125
    k: 0.038
    gravity: 9.79
    yaw_motor_res_speed: 1.6
    tranfer_thresh: 5
    s_bias: 0.0
    z_bias: 0.0
    max_tracking_v_yaw: 6.0
    current_v: 22.0
    delay: 0.01